ONSdigital/response-operations-ui
# Us050 manually change the response status

As a internal user (Survey administrator?)
I need to change the case response status to completed by phone for an
RU for a specific CE for a specific survey
So that the respondent can complete the survey over the phone and they do not receive reminders
    
* Add change link onto reporting-unit.html
* Add change-response-status.html for changing status

Testing
1. Run up ras/rm
1. Run response-operations-ui on branch `us050-manually-change-the-response-status`
1. Run ras-backstage on branch `us050a-change-response-status`
1. Build repo `rm-common-service` branch `us050a-change-response-status-completedbyphone`
1. Build rm-casesvc-api on branch `us050a-change-response-status`
1. Run rm-case-service on branch `us050a-change-response-status-completedbyphone`
1. Load data with `make setup`
1. Open http://localhost:8085/reporting-units/49900000001
1. Click change
1. See updated status

[trello](https://trello.com/c/q5skwt1o/91-us050a-int-manually-change-the-response-status-for-a-given-ru-survey-ce-to-completed-by-phone-xl)
